<snippet>
  <content><![CDATA[
# ${1:Online Supporting Material for Chapter 4}
# Online Supporting Material for Chapter 4
Bacterial genome sequencing and in silico prediction of
Microbial interactions among partners of the tripartite model system

Frederik De Boever
2019


## Tables

__Table 4.1__ - Genome statistics for genomes used in Chapter 4, as derived from CheckM (Parks et al., 2015). Excel .xlsx file is contains three spreadsheets. The first shows genome characteristics of Marinobacter sp. FDB33 and Alteromonas sp. FDB36, sequenced as part of this thesis. The second and third spreadsheets contain a wider set of genomes that were used in the analysis, for the Marinobacter and Alteromonas genera, respectively.

__Table 4.2__ – MAPLE module completion ratio (MCR). Complete list of KEGG module completion ratio’s, represented as the percentage of a module component filled with KEGG Orhology (KO) assigned genes was calculated for FDB33 and FDB36 using KAAS (Moriya et al., 2007) and MAPLE (Takami et al., 2016).
 
__Table 4.3__ – Genome specific KO assigned genes comparing FDB33 and FDB36. KEGG Ortrhology was assigned using KAAS (Moriya et al., 2007) and compared in R. For each genome (FDB33 and FDB36) unique KO identifiers are listed, together with the gene name, functional annotation and respective locus tags.

__Table 4.4__ – CAZY and cellular localisation for FDB33 and FDB36. List of CAZymes for genomes FDB33 and FDB36, with cellular localisation as predicted by SOSUI (Imai et al., 2008), PSORTb (Yu et al., 2010), and SignalP (Petersen et al., 2011).

## Data files

Genome assemblies in fasta format for genomes FDB33 and FDB36
•	Marinobacter_sp_FDB36.fna
•	Alteromonas_sp_FDB33.fna

Genome annotation files in fasta format for FDB33 and FDB36
•	Marinobacter_sp_FDB36.faa
•	Alteromonas_sp_FDB33.faa

## References

Imai, K., Asakawa, N., Tsuji, T., Akazawa, F., Ino, A., Sonoyama, M., et al. (2008). SOSUI-GramN: high performance prediction for sub-cellular localization of proteins in gram-negative bacteria. Bioinformation 2, 417–21. doi:10.6026/97320630002417.

Moriya, Y., Itoh, M., Okuda, S., Yoshizawa, A. C., and Kanehisa, M. (2007). KAAS: An automatic genome annotation and pathway reconstruction server. Nucleic Acids Res. 35. doi:10.1093/nar/gkm321.

Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P., and Tyson, G. W. (2015). CheckM : assessing the quality of microbial genomes recovered from isolates , single cells , and metagenomes. Genome Res. 25, 1043–1055. doi:10.1101/gr.186072.114.Freely.

Petersen, T. N., Brunak, S., von Heijne, G., and Nielsen, H. (2011). SignalP 4.0: discriminating signal peptides from transmembrane regions. Nat. Methods 8, 785–6. doi:10.1038/nmeth.1701.

Takami, H., Taniguchi, T., Arai, W., Takemoto, K., Moriya, Y., and Goto, S. (2016). An automated systemfor evaluation of the potential functionome: MAPLE version 2.1.0. DNA Res. 23, 467–475. doi:10.1093/dnares/dsw030.

Yu, N. Y., Wagner, J. R., Laird, M. R., Melli, G., Rey, S., Lo, R., et al. (2010). PSORTb 3.0: Improved protein subcellular localization prediction with refined localization subcategories and predictive capabilities for all prokaryotes. Bioinformatics 26, 1608–1615. doi:10.1093/bioinformatics/btq249.


]]></content>
  <tabTrigger>readme</tabTrigger>
</snippet>