# Online Supporting Material for Chapter 6
>Functional heterogeneity of _Marinobacter_ links lifestyle and evolutionary history

## Tables

__Table 6.1__ – Isolation Source and Habitat classes of _Marinobacter_ genomes. List of all genomes used in the analysis, with phylogenetic clade, habitat class and geographical coordinates of isolation sites.

__Table 6.2__ – List of Algal-associated Marinobacters derived from literature. Shown are the strain names, isolation source, algal host as well as 16S rRNA sequences used to infer ML phylogeny. References include (Al-Wahaib et al., 2016; Amin et al., 2015; Baker et al., 2016; Brennan, 2014; Le Chevanton et al., 2013; Frazier et al., 2007; Green et al., 2004, 2015; Grossart et al., 2004; Jasti et al., 2005; Kuo and Lin, 2013; Ruh et al., 2009; Seibold et al., 2001; Sher et al., 2011; Tapia et al., 2016) 

__Table 6.3__ – List of _Marinobacter_ type strains.

__Table 6.4__ – Enriched orthologous gene families per phylogenetic clade as identified by Scoary (Brynildsrud et al., 2016).

__Table 6.5__ – Depleted orthologous gene families per phylogenetic clade as identified by Scoary (Brynildsrud et al., 2016).

__Table 6.6__ – Lineage associated orthologous gene families as determined by RandomForest (Liaw and Wiener, 2002), sorted by variable importance.

__Table 6.7__ – Lineage associated orthologous gene families as determined by RandomForest (Liaw and Wiener, 2002), Scoary (Brynildsrud et al., 2016), and Kruskal-Wallis. The list contains those orthologous gene families that were picked up by all three methods.

__Table 6.8__ – Habitat associated gene and domain families as determined by PhyloLM. PhyloLM estimates are given, with Standard Error, p value and FDR-corrected p-values < 0.05. Analysis was run on matrices containing Pfam, CAZY, TCDB and COG.

__Table 6.9__ – Genome statistics for genomes used in Chapter 6, as derived from CheckM (Parks et al., 2015)

__Table 6.10__ – Distribution of Carbohydrate Active Enzyme domains identified in _Marinobacter_ genomes.


## References

Al-Wahaib, D., Al-Bader, D., Al-Shaikh Abdou, D. K., Eliyas, M., and Radwan, S. S. (2016). Consistent Occurrence of Hydrocarbonoclastic Marinobacter Strains in Various Cultures of Picocyanobacteria from the Arabian Gulf: Promising Associations for Biodegradation of Marine Oil Pollution. J. Mol. Microbiol. Biotechnol. 26, 261–268. doi:10.1159/000445686.

Amin, S. A., Hmelo, L. R., Tol, H. M. van, Durham, B. P., Carlson, L. T., Heal, K. R., et al. (2015). Interaction and signalling between a cosmopolitan phytoplankton and associated bacteria. Nature 522, 98–101. doi:10.1038/nature14488.

Baker, L. J., Alegado, R. A., and Kemp, P. F. (2016). Response of diatom-associated bacteria to host growth state, nutrient concentrations, and viral host infection in a model system. Environ. Microbiol. Rep. doi:10.1111/1758-2229.12456.

Brennan, D. (2014). The Microbial Communities of Oleaginous Algae and their Potential to Influence Oil Production.

Brynildsrud, O., Bohlin, J., Scheffer, L., and Eldholm, V. (2016). Rapid scoring of genes in microbial pan-genome-wide association studies with Scoary. Genome Biol. 17. doi:10.1186/s13059-016-1132-8.

Le Chevanton, M., Garnier, M., Bougaran, G., Schreiber, N., Lukomska, E., Bérard, J. B., et al. (2013). Screening and selection of growth-promoting bacteria for Dunaliella cultures. Algal Res. 2, 212–222. doi:10.1016/j.algal.2013.05.003.

Frazier, A. D., Rowe, J. M., Rentz, C. A., Gobler, C. J., and Wilhelm, S. W. (2007). Bacterial lysis of Aureococcus anophagefferens CCMP 1784 (Pelagophyceae). J. Phycol. 43, 461–465. doi:10.1111/j.1529-8817.2007.00353.x.

Green, D. H., Echavarri-bravo, V., Brennan, D., and Hart, M. C. (2015). Bacterial diversity associated with the coccolithophorid algae Emiliania huxleyi and Coccolithus pelagicus f . braarudii Bacterial Diversity Associated with the Coccolithophorid Algae Emiliania huxleyi and Coccolithus pelagicus f . braarudii. 2015.

Green, D. H., Llewellyn, L. E., Negri, A. P., Blackburn, S. I., and Bolch, C. J. S. (2004). Phylogenetic and functional diversity of the cultivable bacterial community associated with the paralytic shellfish poisoning dinoflagellate Gymnodinium catenatum. FEMS Microbiol. Ecol. 47, 345–357. doi:10.1016/S0168-6496(03)00298-8.

Grossart, H. P., Schlingloff, A., Bernhard, M., Simon, M., and Brinkhoff, T. (2004). Antagonistic activity of bacteria isolated from organic aggregates of the German Wadden Sea. FEMS Microbiol. Ecol. 47, 387–396. doi:10.1016/S0168-6496(03)00305-2.

Jasti, S., Sieracki, M. E., Poulton, N. J., Giewat, M. W., and Rooney-Varga, J. N. (2005). Phylogenetic diversity and specificity of bacteria closely associated with Alexandrium spp. and other phytoplankton. Appl. Environ. Microbiol. 71, 3483–3494. doi:10.1128/AEM.71.7.3483-3494.2005.

Kuo, R. C., and Lin, S. (2013). Ectobiotic and Endobiotic Bacteria Associated with Eutreptiella sp. Isolated from Long Island Sound. Protist 164, 60–74. doi:10.1016/j.protis.2012.08.004.

Liaw,  a, and Wiener, M. (2002). Classification and Regression by randomForest. R news 2, 18–22. doi:10.1177/154405910408300516.

Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P., and Tyson, G. W. (2015). CheckM : assessing the quality of microbial genomes recovered from isolates , single cells , and metagenomes. Genome Res. 25, 1043–1055. doi:10.1101/gr.186072.114.Freely.

Ruh, W. W., Ahmad, A., Isa, M. N. M., Mahadi, N. M., Marasan, N. A., and Usup, G. (2009). Diversity of bacteria associated with the benthic marine dinoflagellates Coolia monotis and Ostreopsis ovata from Malaysian waters. J. Sci. Technol. Trop. 5, 23–33.

Seibold, A., Wichels, A., and Schütt, C. (2001). Diversity of endocytic bacteria in the dinoflagellate Noctiluca scintillans. Aquat. Microb. Ecol. 25, 229–235. doi:10.3354/ame025229.

Sher, D., Thompson, J. W., Kashtan, N., Croal, L., and Chisholm, S. W. (2011). Response of Prochlorococcus ecotypes to co-culture with diverse marine bacteria. ISME J. 5, 1125–1132. doi:10.1038/ismej.2011.1.

Tapia, J. E., González, B., Goulitquer, S., Potin, P., and Correa, J. A. (2016). Microbiota Influences Morphology and Reproduction of the Brown Alga Ectocarpus sp. Front. Microbiol. 7, 1–14. doi:10.3389/fmicb.2016.00197.

